src/style.css

```diff
@@ -44,6 +52,13 @@ body {
   border-bottom: 0.5px solid #000;
 }
 
+.strike-through {
+  color: orange;
+  background-color: red;
+}
+
 .clear {
-  text-align: center;
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0 auto;
 }
```

Comment:
Great design @PRATAP-KUMAR 🤝

- [ ] Great job implementing the functionality of the `Clear all is completed` button. However, you haven't implemented the `delete` button. Just in case you want to delete one task. As specified in the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md), you are supposed to implement the delete button functionality.